.github/workflows/kata-runtime-classes-sync.yaml

```diff
@@ -1,11 +1,14 @@
+name: Static checks
+
 on:
   pull_request:
     types:
       - opened
       - edited
       - reopened
       - synchronize
-
+    paths:
+      - 'tools/packaging/kata-deploy/runtimeclasses/**'
```

Comment:
I'm not sure whether it would be useful to add a comment warning people about marking this check required. Because doing so would break things...